### ISSUE TITLE
vite use @vitejs/plugin-legacy plugin will throw ExportDeclaration

### ISSUE THREAD

**Issue by azhuang321** [Menci/vite-plugin-top-level-await#5]
this is code https://github.com/azhuang321/test
```shell
PS E:\Project\test> yarn build
yarn run v1.22.19
$ vite build
vite v2.9.13 building for production...
✓ 10 modules transformed.
dist/assets/polyfills-legacy.5448393d.js   43.61 KiB / gzip: 18.46 KiB
dist/assets/index-legacy.d602fd2f.js       48.29 KiB / gzip: 19.04 KiB
rendering chunks (1)...[vite-plugin-top-level-await] Unexpected top-level statement "ExportDeclaration" in Rollup's output chunk. Please open an issue at https://github.com/Menci/vite-plugin-top-level-await/issues.
error during build:
Error: Unexpected top-level statement "ExportDeclaration" in Rollup's output chunk. Please open an issue at https://github.com/Menci/vite-plugin-top-level-await/issues.
    at raiseUnexpectedNode (E:\Project\test\node_modules\vite-plugin-top-level-await\dist\utils\error.js:6:11)
    at E:\Project\test\node_modules\vite-plugin-top-level-await\dist\transform.js:36:49
    at Array.filter (<anonymous>)
    at transformModule (E:\Project\test\node_modules\vite-plugin-top-level-await\dist\transform.js:26:30)
    at E:\Project\test\node_modules\vite-plugin-top-level-await\dist\index.js:64:64
    at Array.map (<anonymous>)
    at Object.generateBundle (E:\Project\test\node_modules\vite-plugin-top-level-await\dist\index.js:61:57)
    at async Bundle.generate (E:\Project\test\node_modules\rollup\dist\shared\rollup.js:16116:9)
    at async E:\Project\test\node_modules\rollup\dist\shared\rollup.js:23732:27
    at async catchUnfinishedHookActions (E:\Project\test\node_modules\rollup\dist\shared\rollup.js:23174:20)
error Command failed with exit code 1.
```

**Comment by Menci** [comment]
Fixed in [v1.0.1](https://github.com/Menci/vite-plugin-top-level-await/releases/tag/v1.0.1).

**Comment by azhuang321** [comment]
> Fixed in [v1.0.1](https://github.com/Menci/vite-plugin-top-level-await/releases/tag/v1.0.1).

thinks
